package.json

```diff
@@ -16,43 +16,42 @@
   "files": [
     "dist/*"
   ],
-  "module": "dist/browser/index.mjs",
-  "main": "dist/node/index.js",
-  "typings": "dist/node/index.d.ts",
-  "browser": "./dist/browser/index.mjs",
+  "module": "dist/esm/mod.js",
+  "main": "dist/cjs/index.js",
+  "typings": "./dist/cjs/mod.d.ts",
+  "browser": "./dist/browser/mod.js",
   "exports": {
-    ".": {
+    "bun": "./dist/bun/mod.js",
+    "node": {
       "import": {
-        "browser": "./dist/browser/index.mjs",
-        "types": "./dist/browser/index.d.ts"
+        "types": "./dist/esm/mod.d.ts",
+        "default": "./dist/esm/mod.js"
       },
       "require": {
-        "node": "./dist/node/index.js",
-        "types": "./dist/node/index.d.ts"
+        "types": "./dist/cjs/mod.d.ts",
+        "default": "./dist/cjs/index.js"
       }
+    },
+    "default": {
+      "types": "./dist/browser/mod.d.ts",
+      "default": "./dist/browser/mod.js"
     }
   },
   "author": "The Extism Authors <oss@extism.org>",
   "license": "BSD-3-Clause",
   "devDependencies": {
-    "@types/jest": "^29.2.2",
-    "@types/sync-fetch": "^0.4.0",
+    "@bjorn3/browser_wasi_shim": "^0.2.14",
+    "@playwright/test": "^1.39.0",
+    "@types/node": "^20.8.7",
+    "@typescript-eslint/eslint-plugin": "^6.8.0",
+    "@typescript-eslint/parser": "^6.8.0",
     "esbuild": "^0.15.13",
-    "esbuild-jest": "^0.5.0",
-    "jest": "^29.6.4",
-    "jest-environment-jsdom": "^29.3.1",
+    "esbuild-node-builtin": "^0.1.1",
+    "eslint": "^8.51.0",
+    "playwright": "^1.39.0",
     "prettier": "^2.7.1",
-    "ts-jest": "^29.0.3",
-    "tslint": "^6.1.3",
-    "tslint-config-prettier": "^1.18.0",
-    "typedoc": "^0.23.20",
-    "typescript": "^4.8.4"
-  },
-  "dependencies": {
```

Comment:
Is it really true that we have no more external runtime dependencies?

Reply:
It's true! Even after adding minimatch, Node has no external runtime deps.